specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2023-04-01-preview/examples/CreateOrUpdateTest.json

```diff
@@ -9,7 +9,17 @@
       "displayName": "Performance_LoadTest",
       "loadTestConfiguration": {
         "engineInstances": 6,
-        "splitAllCSVs": true
+        "splitAllCSVs": true,
```

Comment:
regional config is a test property, and can't be modified for a test run.
But should be also return regional config in the response of test run, else with a given test run object, the regional config wont be deducable, even though test run was multiregion

Reply:
yes will update example. it is readonly for testruns